test/integration/fixtures/blocks/core__comments.serialized.html

```diff
@@ -1,5 +1,5 @@
-<!-- wp:comments-query-loop -->
-<div class="wp-block-comments-query-loop"><!-- wp:comments-title {"level":4,"borderColor":"vivid-red","backgroundColor":"primary","textColor":"background","fontSize":"large","style":{"spacing":{"padding":{"top":"6px","right":"6px","bottom":"6px","left":"6px"}},"border":{"width":"3px","radius":"100px"}}} /-->
+<!-- wp:comments -->
+<div class="wp-block-comments"><!-- wp:comments-title /-->
```

Comment:
Will this change affect anyone that relied on the class for giving styles? What is our approach in similar issues?

Reply:
I think we need to keep the old class name around, unfortunately 😬